EventFilter/ESRawToDigi/src/ESUnpacker.cc

```diff
@@ -208,16 +208,31 @@ void ESUnpacker::interpretRawData(int fedId, const FEDRawData & rawData, ESRawDa
   ESDCCHeader.setOptoRX1(optoRX1_ + dccCRC2_);
   ESDCCHeader.setOptoRX2(optoRX2_ + dccCRC3_);
   ESDCCHeader.setFEChannelStatus(FEch_status);
+  int enableOptoRX[3] = {-1, -1, -1};
+  int NenableOptoRX = 0; 
+  if (optoRX0_ == 128) {
+    enableOptoRX[NenableOptoRX] = 0;
+    NenableOptoRX++;
+  }
+  if (optoRX1_ == 128) {
+    enableOptoRX[NenableOptoRX] = 1;
+    NenableOptoRX++;
+  }
+  if (optoRX2_ == 128) {
+    enableOptoRX[NenableOptoRX] = 2;
+    NenableOptoRX++;
```

Comment:
NenableOptoRX is not used after this line, so the increment would seem to serve no purpose. Please delete it.